orbit/cmd/orbit/orbit.go

```diff
@@ -614,10 +614,14 @@ func main() {
 
 		// create the notifications middleware that wraps the orbit client
 		// (must be shared by all runners that use a ConfigFetcher).
-		const renewEnrollmentProfileCommandFrequency = time.Hour
+		const (
+			renewEnrollmentProfileCommandFrequency = time.Hour
+			windowsMDMEnrollmentCommandFrequency   = time.Hour
```

Comment:
QQ: Would this mean that once MDM for Windows is enabled on the server, the hosts could take up to an hour to perform the enrollment? This is OK to me, we might need to document it on issue #12260